Added map data output information

Dumanbruh opened the pull request.

Dumanbruh committed:
Refactor map component to display air sensor data and show info window on circle click

Commit:
Refactor air sensor signal interface and custom map component

Dumanbruh committed:
Refactor geolocation handling in AppLayout and CustomMap components

Dumanbruh committed:
Refactor getFullDateWithTime function to format date and time

Dumanbruh committed:
Refactor CSS for Google Maps info window and base styles

Dumanbruh committed:
Refactor page title and description in index.html

talgatulyzh commented on src/common/getFullDate.ts:
```diff
@@ -0,0 +1,7 @@
+export const getFullDateWithTime = (val: string): string => {
```
Можно в date-fns форматировать время  в удобный формат (по желанию)

Dumanbruh replied:
Поменял(по своему желанию)

talgatulyzh reviewed (approved):
lgtm

Dumanbruh committed:
Refactor map component to handle camera changes and initial position

Dumanbruh committed:
Refactor getFullDateWithTime function to use date-fns library for date and time formatting

talgatulyzh commented:
![go-go-go-toby-stephens.gif](https://github.com/user-attachments/assets/82f73ca8-34fc-4f99-8253-802d1615b991)

Dumanbruh committed:
Refactor mapUtils.tsx to use date-fns library for date and time formatting
Refactor routes.tsx to import AppLayout from the correct location
Move AppLayout.tsx to the pages/home directory
Update CustomMap.tsx to handle camera changes and display info window
Remove SensorCircle.tsx from the project

Dumanbruh committed:
Refactor import statement in custom-map.tsx to remove unused RenderingType import
